### ISSUE TITLE
JAWS is not announcing the label name "Input type" when radio buttons are receiving screen reader focus: A11y_MySQL extension for Azure Data Studio_Connect to a MySQL Server_ScreenReader

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio-mysql#236]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM_MySQL extension for Azure Data Studio_Win32_Mar2024; #MySQL extension for Azure Data Studio; #Win32; #Benchmark; #Win11; #JAWS; #ScreenReader; #WCAG1.3.1; #InfoandRelationships; #AILimited; #Element:Label; #FTP;

### Environment and OS details:
Azure Data Studio Version: 1.48.0 (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296
Screen Reader: JAWS 2024, NVDA 2023.3.2

### Repro Steps:

1. Launch Azure Data Studio.
2. Start JAWS.
3. Tab to the new connection button and press enter.
4. Tab to "Input type" section radio buttons.
5. Observe JAWS is not announcing label name when radio buttons are receiving screen reader focus.

### Actual Result:
JAWS is not announcing the label name "Input type" when radio buttons are receiving screen reader focus.

**Note: This issue does not repro with NVDA.**

### Expected Result:
JAWS should be announcing the label name "Input type" when radio buttons are receiving screen reader focus.

### User Impact:
Screen reader users will not be able to understand the purpose of the radio buttons and will get confused.

### Attachment:
[JAWS_Repro.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14718888/JAWS_Repro.zip)
[NVDA_NotRepro.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14718891/NVDA_NotRepro.zip)
Code snippet:
![Screenshot (1614)](https://github.com/microsoft/azuredatastudio-mysql/assets/86710888/78755db0-7d36-47d8-af07-f943828038a8)

**Comment by nitishsharma25** [comment]
Hi @Swati1700, for our extension, under Connection type, you have to select MySQL, Input type field is external to MySQL as you can see here:
![image](https://github.com/microsoft/azuredatastudio-mysql/assets/112425736/7c2e3783-f591-4960-a25d-192277cf2cd8)

**Comment by Swati1700** [comment]
As per the above comment For MYSQL connection type issue does not repro, hence closing the bug.
